experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_ansible_logic.yml

```diff
@@ -0,0 +1,213 @@
+      - block:
+          - name: Construct chaos result name (experiment_name)
+            set_fact:
+              c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
+            when: lookup('env','CHAOSENGINE')
+
+          - include_tasks: /utils/runtime/update_chaos_result_resource.yml
+            vars:
+              status: 'SOT'
+              namespace: "{{ a_ns }}"
+
+      # DISPLAY APP INFORMATION
+      - name: Display the app information passed via the test job
+        debug:
+          msg:
+            - "The application info is as follows:"
+            - "Namespace    : {{ a_ns }}"
+            - "Label        : {{ a_label }}"
+
+      - block: 
+        # Gcloud authentication
+        - name: Gcloud authentication
+          include_tasks: "/utils/cloud/gcp/gcloud_configure.yml"
+          when: "cloud_platform == 'GCP'"
+        
+        # AWS authentication
+        - name: AWS authentication
+          include_tasks: "/utils/cloud/aws/aws_configure.yml"
+          when: "cloud_platform == 'AWS'"
+
+      ## PRE-CHAOS DISK LIVENESS CHECK
+      - block:
+        - name: Verify that the disk is connected to node (pre)
+          include_tasks: "/utils/cloud/gcp/status_disk.yml"
+          when: "cloud_platform == 'GCP'"
+
+        - name: Verify that the disk is connected to node (pre)
+          include_tasks: "/utils/cloud/aws/status_disk.yml"
+          when: "cloud_platform == 'AWS'"
+
+        # Checking disk is attached to node
+        - debug: 
+            msg: "specified disk is attached to node"   
+          when: "inuse is defined and inuse == true"
+
+        - fail:
+            msg: "specified disk not attached to node"
+          when: "inuse is defined and inuse == false"
+        
+      - name: Verify that the AUT (Application Under Test) is running
+        include_tasks: "/utils/common/status_app_pod.yml"
+        vars:
+          app_ns: "{{ a_ns }}"
+          app_label: "{{ a_label }}"      
+          delay: 5
+          retries: 60
+
+      # Auxiliary application health check status
+      - block:
+
+        - name: Record auxiliary appinfo 
+          set_fact:
+            auxiliary_appinfo_list: "{{ auxiliary_appinfo.split(',') }}"
+
+        - include: /utils/common/status_app_pod.yml
+          vars:
+            app_ns: "{{ item.split(':')[0] }}"
+            app_label: "{{ item.split(':')[1] }}"
+            delay: 5
+            retries: 60
+          with_items:
+            - "{{ auxiliary_appinfo_list }}" 
+
+        when: auxiliary_appinfo is defined and auxiliary_appinfo != ''
+
+      - block:
+        - name: Verify that cStor pool is healthy
+          shell: >
+            kubectl get csp -n {{ openebs_ns }} -o jsonpath='{.items[*].status.phase}'
+          register: csp_status
+          until: "((csp_status.stdout_lines | unique) | length) == 1 and 'Healthy' in csp_status.stdout"
+          delay: 5
+          retries: 60
```

Comment:
delay: 2 and retries: 90 and pls add this task to separate utils. As it is repeated in all pool experiments